### PR TITLE
Fix permission error and allow interactive mode in entrypoint.sh

### LINKED ISSUE #1: Container startup error
```
opt/fahclient/entrypoint.sh\": permission denied": unknown

ERROR: for folding-at-home  Cannot start service folding-at-home: OCI runtime create failed: container_linux.go:349: starting container process caused "exec: \"/opt/fahclient/entrypoint.sh\": permission denied": unknown
ERROR: Encountered errors while bringing up the project.

```

### PR DESCRIPTION
**Pull request by sndrtj**
Hello there,

This should fix #1 and allows interactive mode in the entrypoint.sh which can be useful for e.g. debugging.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -18,7 +18,9 @@ RUN useradd --system folding && \
     apt-get autoremove --yes && \
     rm -rf /var/lib/apt/lists/*
 
-COPY entrypoint.sh /opt/fahclient
+COPY --chown=folding:folding entrypoint.sh /opt/fahclient
```
Comment (yurinnick) [inline Dockerfile:21]: That's cool, I didn't know about it!

File: entrypoint.sh [hunk 2]
```diff
@@ -1,11 +1,20 @@
...
-    --team=${TEAM} \
-    --passkey=${PASSKEY} \
-    --gpu=${ENABLE_GPU} \
-    --smp=${ENABLE_SMP} \
-    --power=full \
-    --gui-enabled=false \
-    ${@}
+# if a command was specified, we run that command. E.g. helpful when we do
+# docker run -it <container> /bin/bash
+if [ "${1}" ]
```
Comment (yurinnick) [inline entrypoint.sh]: Isn't it gonna be triggered by `docker run <image> --some-random-arg=value` as well?

**Review by yurinnick — CHANGES_REQUESTED** [review]
Please remove if/else branching in favor of `docker run --entrypoint=/bin/bash <image>`

File: entrypoint.sh (hunk 2 above)
Comment (sndrtj) [reply entrypoint.sh]: :+1: I hadn't considered that.

**Review by yurinnick — APPROVED** [review]

**Comment by yurinnick** [comment]
Thanks a lot for your contribution!

**Pull request merged**